lib/v2/hdu/cs/index.js

```diff
@@ -0,0 +1,64 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+
+const link = 'https://computer.hdu.edu.cn';
+const host = 'https://computer.hdu.edu.cn/6738/list.htm';
+
+const getSingleRecord = async () => {
+    const res = await got({
+        method: 'get',
+        url: host,
+    });
+
+    const $ = cheerio.load(res.data);
+    const list = $('.posts-list').find('li');
+
+    return (
+        list &&
+        list
+            .map((index, item) => {
+                item = $(item);
+                const dateTxt = item.find('.date').text();
+                const date = dateTxt.slice(1, dateTxt.length - 1);
+                return {
+                    title: item.find('a').text(),
+                    pubDate: new Date(date),
+                    link: item.find('a').attr('href'),
+                };
+            })
+            .get()
+    );
+};
+
+module.exports = async (ctx) => {
+    const items = await getSingleRecord();
+    const out = await Promise.all(
+        items.map(async (item) => {
+            const url = link + item.link;
```

Comment:
You can return a concatenated link
```suggestion
                    link: link + item.find('a').attr('href'),
                };
            })
            .get()
    );
};

module.exports = async (ctx) => {
    const items = await getSingleRecord();
    const out = await Promise.all(
        items.map(async (item) => {
```